dist/src/main/resources/application.properties

```diff
@@ -14,3 +14,4 @@ management.endpoint.loggers.enabled=true
 #Allow partitions api - this allows to trigger snapshots and pause processing
 management.endpoint.partitions.enabled=true
 
+zeebe.broker.network.host=::1
```

Comment:
This will default to bind to ipv6 loopback always - what if we deploy to a VM without IPv6 support? I know it's unlikely, but I wouldn't set it as default. We can set it in the environment variables in the test.